shopify_transporter.gemspec

```diff
@@ -33,4 +33,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'activesupport', '~> 5.1'
   spec.add_dependency 'thor', '~> 0.20'
   spec.add_dependency 'yajl-ruby', '~> 1.3'
+  spec.add_dependency 'sequel', '~> 5.12'
+  spec.add_dependency 'mysql2'
+  spec.add_dependency 'pg'
```

Comment:
I think we can stick with just mysql: https://magento.stackexchange.com/questions/30969/magento-using-postgres